orix/plot/stereographic_plot.py

```diff
@@ -167,6 +169,90 @@ def plot(self, *args, **kwargs):
 
         super().plot(x, y, **updated_kwargs)
 
+    def pole_density_function(
+        self,
+        azimuth: np.ndarray,
+        polar: np.ndarray,
+        resolution: float = 1,
+        sigma: float = 5,
+        log: bool = False,
+        colorbar: bool = True,
+        **kwargs,
+    ):
+        """Compute the Pole Density Function (PDF) of vectors in the
+        stereographic projection.
+
+        Parameters
+        ----------
+        azimuth, polar
+            Azimuth and polar angles in radians.
```

Comment:
```suggestion
            Azimuth and polar angles of the vectors in radians.
```

... or something similar.